src/applications/accredited-representative-portal/components/common/Header/UserNav.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom-v5-compat';
+import UserNavLinks from './UserNavLinks';
+import NavigationDropdown from './NavigationDropdown';
+import {
+  selectUserProfile,
+  selectIsUserLoading,
+} from '../../../selectors/user';
+import { SIGN_IN_URL } from '../../../constants';
+
+const UserNav = ({ isMobile }) => {
+  const profile = useSelector(selectUserProfile);
+  const isLoading = useSelector(selectIsUserLoading);
+
+  let content;
+
+  if (isLoading) {
+    content = (
+      <div className="loading-icon-container">
+        <va-loading-indicator
+          data-testid="user-nav-loading-icon"
+          label="Loading"
+        />
+      </div>
+    );
+  } else if (!profile) {
+    content = (
+      <Link
+        data-testid="user-nav-sign-in-link"
+        className="usa-button usa-button-primary nav__sign-in"
+        to={SIGN_IN_URL}
+      >
+        Sign in
+      </Link>
```

Comment:
This links to a new single page application at the fully qualified URL given by `SIGN_IN_URL`, so the relative navigation that `react-router` provides with its `Link` component won't work here, and indeed this `Link` doesn't work. This can be changed back to an anchor element instead.